pallene/checker.lua

```diff
@@ -11,7 +11,7 @@ local symtab = require "pallene.symtab"
 local types = require "pallene.types"
 local typedecl = require "pallene.typedecl"
 local util = require "pallene.util"
-local builtin_modules = require "pallene.builtin_modules"
+local inspect = require "inspect"
```

Comment:
Tip: you can require inspect in the same line where you use it. This way you don't forget about the `require "inspect"` at the top of the file after you delete the debugging print statement.

     print(require("inspect")(something))

Reply:
Alright. Will make the changes.